ci/docs/layout.html

```diff
@@ -12,4 +12,36 @@
   ga('send', 'pageview');
 
 </script>
+
+{# check current version against supported-releases.json #}
+<script>
+$( document ).ready( function() {
+
+  // gets version number minus whitespace and bugfix version
+  var $version_number = $('.version').text().trim().split('.', 2).join('.')
+
+  var supported_versions = []
+  // get JSON from server and load supported_versions
+  $.getJSON("https://raw.githubusercontent.com/pulp/pulp_packaging/master/ci/config/releases/supported-releases.json" , function(data) {
```

Comment:
I would rewrite this to be: 

```
$( document ).ready( function() {
	var $version_number = $('.version').text().trim().split('.', 2).join('.')
	
	// Declared outside of the ajax promise since it does not change
	var warning = '\
        <div class="admonition warning"> \
            <p class="first admonition-title">Warning</p> \
            <p class="last">You are looking at the docs for an unsupported version of Pulp. \
             Consider upgrading to a  \
             <a href="http://pulpproject.org/docs/">supported version</a></p> \
        </div>'

	// this is not necessary since the returned value is already an array
	// var supported_versions = []


	$.getJSON("https://raw.githubusercontent.com/pulp/pulp_packaging/master/ci/config/releases/supported-releases.json" , function(data) {
	// this is technically the success callback so no need for $.done() (I need to read docs more carefully)
		if ($.inArray($version_number, data) == -1) {
			$('.rst-content').prepend(warning)
		}
	})
	
})
```